runtime/src/iree/vm/stack.c

```diff
@@ -230,13 +230,24 @@ IREE_API_EXPORT iree_status_t iree_vm_stack_initialize(
   return iree_ok_status();
 }
 
-IREE_API_EXPORT void iree_vm_stack_deinitialize(iree_vm_stack_t* stack) {
+IREE_API_EXPORT void iree_vm_stack_reset(iree_vm_stack_t* stack) {
```

Comment:
What is the relationship between `iree_vm_stack_reset` parts and other parts of this PR?

Reply:
Cleanup as part of what I was doing.